apps/emqx_conf/src/emqx_conf_app.erl

```diff
@@ -152,11 +152,11 @@ copy_override_conf_from_core_node() ->
                 _ ->
                     [{ok, Info} | _] = lists:sort(fun conf_sort/2, Ready),
                     #{node := Node, conf := RawOverrideConf, tnx_id := TnxId} = Info,
-                    Msg = #{
+                    ?SLOG(debug, #{
                         msg => "copy_overide_conf_from_core_node_success",
-                        node => Node
-                    },
-                    ?SLOG(debug, Msg),
+                        node => Node,
+                        envs => application:get_all_env(emqx)
```

Comment:
this is a lot of config and running in risk of dumping secrets to logs.

Reply:
not at all, emqx's env only have 
```
[{init_config_load_done, boolean()},
 {local_override_conf_file,"./data2/configs/local-override.conf"},
 {cluster_override_conf_file,"./data2/configs/cluster-override.conf"},
 {config_files,["/Users/zhongwen/github/emqx/emqx-v5/_build/emqx/rel/emqx/etc/emqx.conf"]},
 {data_dir,"./data2"},
 {cluster_rpc_init_tnx_id,1}]
```
before init.

Reply:
DONE